tests/cypress/e2e/llamalend/llamalend-markets.cy.ts

```diff
@@ -127,49 +129,52 @@ describe(`LlamaLend Markets`, () => {
 
   it(`should allow filtering by using a slider`, () => {
     const [columnId, initialFilterText] = oneOf(
-      ['liquidityUsd', 'Liquidity: $0 -'],
-      ['tvl', 'TVL: $10k -'],
-      ['utilizationPercent', 'Utilization: 0% -'],
+      ['liquidityUsd', '$0 -'],
+      ['tvl', '$10k -'],
+      ['utilizationPercent', '0% -'],
     )
     cy.viewport(1200, 800) // use fixed viewport to have consistent slider width
     cy.get(`[data-testid^="data-table-row"]`).then(({ length }) => {
       cy.get(`[data-testid="minimum-slider-filter-${columnId}"]`).should('not.be.visible')
       cy.get(`[data-testid="btn-expand-filters"]`).click({ waitForAnimations: true })
       cy.get(`[data-testid="minimum-slider-filter-${columnId}"]`).should('contain', initialFilterText)
       cy.get(`[data-testid="slider-${columnId}"]`).should('not.exist')
-      cy.get(`[data-testid="minimum-slider-filter-${columnId}"]`).click()
+      cy.get(`[data-testid="minimum-slider-filter-${columnId}"]`).click({ waitForAnimations: true })
       /**
        * Using `force: true` to bypass Cypress' element visibility check.
        * The slider may have pseudo-elements that interfere with Cypress' ability to interact with it.
        * We've tried alternative approaches (adding waits, adjusting click coordinates) but they didn't resolve the issue.
        * The application behavior works correctly despite this test accommodation.
        */
-      cy.get(`[data-testid="slider-${columnId}"]`).click(60, 20, { force: true })
+
+      // Wait 1 second for the slider to appear after the click
+      cy.wait(1000) // Wait 1 seconds for the UI to update after slider interaction
+      cy.get(`[data-testid="slider-${columnId}"]`).click(60, 10, { force: true })
```

Comment:
Yeah I know testing with transitions is a pain in the butt, but please never use `wait` with a constant time. It always waits too much or too little, making the test too slow sometimes and to break other times.

I think the right way is to use [`onTransitionExited`](https://mui.com/material-ui/api/modal/#modal-prop-onTransitionExited) to add a test id to the drawer. Then we can add that testId here.

```suggestion
      cy.get(`[data-testid="drawer-open"] [data-testid="slider-${columnId}"]`).click(60, 10, { force: true })
```